api/nodes/views.py

```diff
@@ -1282,6 +1282,8 @@ class NodeFilesList(generics.ListAPIView, WaterButlerMixin, ListFilterMixin, Nod
         base_permissions.TokenHasScope,
     )
 
+    ordering = ('materialized_path',)  # default ordering
```

Comment:
As discussed on Flowdock, let's sort by `date_modified` for now so that we can make files consistent with nodes, which will also soon be sorted by `date_modified`.